src/components/tab-bar/tab-bar.tsx

```
@@ -119,25 +121,25 @@ export class TabBar {
                     <div class="scroll-fade left" />
                     <div class="scroll-button left">
-                        <limel-icon-button
-                            icon="angle_left"
-                            elevated={true}
+                        <button
                             tabindex="-1"
                             aria-hidden="true"
                             disabled={!this.canScrollLeft}
                             onClick={this.handleLeftScrollClick}
-                        />
+                        >
```

Biome (1.9.4)

[error] Disallow aria-hidden="true" from being set on focusable elements.

aria-hidden should not be set to true on focusable elements because this can lead to confusing behavior for screen reader users.
Unsafe fix: Remove the aria-hidden attribute from the element.


(lint/a11y/noAriaHiddenOnFocusable)